apps/dolly-frontend/src/main/js/src/components/bestillingsveileder/stegVelger/steg/steg1/paneler/NavAnsatt.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import Panel from '~/components/ui/panel/Panel'
+import { Attributt, AttributtKategori } from '../Attributt'
+import _get from 'lodash/get'
+
+export const NavAnsattPanel = ({ stateModifier }: any) => {
+	const sm = stateModifier(NavAnsattPanel.initialValues)
+	return (
+		// @ts-ignore
+		<Panel
+			heading={NavAnsattPanel.heading}
+			checkAttributeArray={sm.batchAdd}
+			uncheckAttributeArray={sm.batchRemove}
+			iconType="organisasjon"
```

Comment:
Kanskje vurdere et annet ikon? Passer forsåvidt til "organisasjonsmaster", men det navnet blir vel ikke vist for brukeren? Kunne vurdert noe som går mer på "ansatt" evt. NAV-logo 🤔